rskj-core/src/main/java/co/rsk/rpc/modules/personal/PersonalModuleWalletEnabled.java

```diff
@@ -123,6 +123,9 @@ public String newAccount(String passphrase) {
     public String importRawKey(String key, String passphrase) {
         String s = null;
         try {
+            if (key != null && key.startsWith("0x")) {
```

Comment:
Why isn't there a `Hex.decode` method that supports starting with `Ox`? It would save adding this so many times over all the codebase...

Reply:
Hahaha, that could be nice, maybe we can add a wrapper some day and use the wrapper instead of this. 🤔